web-app/src/games/games.controller.ts

```diff
@@ -26,16 +26,21 @@ export class GamesController {
   @Post('')
   @Redirect('/')
   async createNewGame(
-    @Body() gameParams: { size?: number; fromFile?: boolean },
+    @Body()
+    gameParams: { size?: number; fromFile?: boolean; soloMode?: string },
     @Req() req: Request,
   ) {
     let newGame: Game;
     if (gameParams.fromFile) {
-      newGame = await this.gameService.createNewGameFromFile(req.sessionID);
+      newGame = await this.gameService.createNewGameFromFile(
+        req.sessionID,
+        gameParams.soloMode === 'on',
+      );
     } else {
       newGame = await this.gameService.createNewGame(
         gameParams.size,
         req.sessionID,
+        gameParams.soloMode === 'on',
```

Comment:
If we keep the string value we can at least calculate it only one time at the top of the function and use the variable elsewhere.